sfa_dash/form_utils/converters.py

```diff
@@ -436,6 +564,13 @@ def formdata_to_payload(cls, form_dict):
         report_params['end'] = form_dict['period-end']
         report_params = cls.apply_crps(report_params)
         report_dict = {'report_parameters': report_params}
+        if len(costs) > 0:
+            cost_name = costs[0]['name']
+            report_params['metrics'].append('cost')
```

Comment:
I guess check if added first. Although it should probably get checked automatically on the dashboard so it's not opaque to the user